fastlane/lib/fastlane/actions/commit_github_file.rb

```diff
@@ -153,7 +153,7 @@ def self.return_value
         [
           "A hash containing all relevant information for this commit",
           "Access things like 'html_url', 'sha', 'message'"
-        ].join("\n")
+        ].join("<br>")
```

Comment:
This needs to stay a `\n` for terminal output 🙃 

![screen shot 2018-04-10 at 5 34 17 am](https://user-images.githubusercontent.com/401294/38552175-f2b1b522-3c80-11e8-954c-a291492a3ea0.png)

Reply:
Damn :( Thanks for catching this! This means I need to `gsub` `\n` with `<br>` in the `erb`, right? I'll test it tonight.